Assignment-8-August-12/gallery/src/App.js

```diff
@@ -0,0 +1,12 @@
+
+import './App.css';
+import Upload from './upload';
+function App() {
+  return (
+    <div className="App">
+      <Upload></Upload>
```

Comment:
use closing element like this

```suggestion
      <Upload />
```